bndtools.core/bndtools.shared.bndrun

```diff
@@ -72,8 +87,6 @@ crossplatform.bundles: \
 	javax.inject,\
 	javax.xml,\
 	javax.xml.stream,\
-	slf4j.api,\
-	slf4j.simple,\
```

Comment:
I added a sil4j impl that is required by m2e bundle, see here: https://github.com/gamerson/bndtools/blob/3c91c1cab672469e9991e9e9335beaceb080c4eb/bndtools.core/bndtools.shared.bndrun#L68  I was thinking that would be enough, maybe we can remove that one and put slf4j.simple back?